strings.md

```diff
@@ -1009,6 +1010,31 @@ With `~2,2f`:
 
 And we're happy with this result.
 
+### Iteration
+Create a string from a list with iteration construct `~{str~}`:
```

Comment:
I am a bit confused about "str" in here. What would be best? It is a format directive. Use some `<directive>` placeholder (not best), `…` or nothing?

Reply:
I don't know what is the best in this case.

The "str" here is actually copied from documentation in LispWorks HyperSpec.